.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+---
+
+## Checklist:
+
+- Does this pull request close an issue? We encourage you to open an issue first if this pull request (PR) is not a
+  minor change.
+  - [ ] No
+    - [ ] The change in this pull request is minor.
+  - [ ] Yes
+    - Close #{issue_number}
+
+For the following questions, only check the boxes that are applicable.
+
+- [ ] Add descriptive pull request text.
```

Comment:
Can we add a section for this like `## PR Description:` below the `##Checklist:` section?

Reply:
oh I just saw your earlier comment about adding the line above `## Checklist:`. Maybe add `## PR Description:` above that line then?

Reply:
When you open new PR, the commit message should have been automatically inserted to the top. That's why I didn't put an additional section.

Reply:
I updated the text. See if this would clear some confusion

Reply:
Oh ok gotcha. The text helps thanks. For me the commit message always goes into the PR title, but the description stays blank. I'll have to look into how Github handles this then.